src/main/java/wallet/thread/MyThread.java

```diff
@@ -0,0 +1,74 @@
+package wallet.thread;
+
+import wallet.logic.LogicManager;
+import wallet.model.record.Loan;
+import wallet.model.record.LoanList;
+import wallet.ui.Ui;
+
+public class MyThread implements Runnable {
+
+    private boolean autoRemind;
+    private Thread thread;
+    private int timeInSeconds;
+    private int counter;
+    private LoanList loanList;
+    private Ui ui;
+
+    /**
+     * Constructs a custom thread.
+     *
+     * @param loanList      The LoanList object.
+     * @param timeInSeconds The time in seconds.
+     */
+    public MyThread(boolean autoRemind, LoanList loanList, int timeInSeconds) {
+
+        this.autoRemind = autoRemind;
+        this.timeInSeconds = timeInSeconds;
+        this.loanList = loanList;
+        this.ui = new Ui();
+        thread = new Thread(this);
+        thread.start(); // Starting the thread
+    }
+
+
+    /**
+     * Executes the thread.
+     */
```

Comment:
Unnecessary comment that does not provide any useful information. Perhaps state what the thread does instead.